### PR TITLE
Dfr 2916 general orders not going into approved orders folder

### PR DESCRIPTION
**Pull request by AbuSalamHMCTS**
https://tools.hmcts.net/jira/browse/DFR-2916

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by hamzahtahirhmcts — DISMISSED** [review]

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/HearingOrderControllerTest.java [hunk 1]
```diff
@@ -29,6 +30,7 @@
 import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.DRAFT_DIRECTION_DETAILS_COLLECTION_RO;
 import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.LATEST_DRAFT_DIRECTION_ORDER;
 
+@Ignore
```
Comment (jthmcts) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/HearingOrderControllerTest.java:33]: Remove this or add a comment as to why it can't be fixed

File: build.gradle [hunk 2]
```diff
@@ -203,7 +203,12 @@ sonarqube {
                            '**/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/GenericAboutToStartOrSubmitCallbackResponse.java',
                            '**/uk/gov/hmcts/reform/finrem/caseorchestration/utils/csv/CaseReference.java',
                            '**/uk/gov/hmcts/reform/finrem/caseorchestration/utils/csv/CaseReferenceKeyValue.java',
-                            '**/uk/gov/hmcts/reform/finrem/caseorchestration/service/SystemUserService.java'
+                            '**/uk/gov/hmcts/reform/finrem/caseorchestration/service/SystemUserService.java',
```
Comment (jthmcts) [inline build.gradle:206]: This all need fixing. We shouldn't be adding exceptions to sonar without good reason

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/HearingOrderController.java [hunk 3]
```diff
@@ -113,6 +123,7 @@ public ResponseEntity<AboutToStartOrSubmitCallbackResponse> startHearingOrderApp
             caseData.remove(LATEST_DRAFT_DIRECTION_ORDER);
         }
 
+
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/HearingOrderController.java:126]: Unnecessary whitespace

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/documentcatergory/UploadGeneralDocumentsCategoriser.java [hunk 4]
```diff
@@ -26,6 +27,8 @@ public class UploadGeneralDocumentsCategoriser extends DocumentCategoriser {
     private static final List<UploadGeneralDocumentType> RESPONDENT_DOC_TYPES = List.of(LETTER_EMAIL_FROM_RESPONDENT,
         LETTER_EMAIL_FROM_RESPONDENT_SOLICITOR, LETTER_EMAIL_FROM_RESPONDENT_CONTESTED);
 
+    private static final List<UploadGeneralDocumentType> DRAFT_ORDER_DOC = List.of(DRAFT_ORDER);
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/documentcatergory/UploadGeneralDocumentsCategoriser.java:30]: This isn't required. Just reference DRAFT_ORDER where it is needed below

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/documentcatergory/UploadedDraftOrderCategoriser.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DraftDirectionOrder;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DraftDirectionOrderCollection;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.document.DocumentCategory;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.FeatureToggleService;
+
+import java.util.List;
+import java.util.Optional;
+
+@Configuration
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/documentcatergory/UploadedDraftOrderCategoriser.java:13]: @Component not @Configuration

**Review by jamiephillipshmcts — DISMISSED** [review]
As discussed, check locally on master- create new case and run upload draft order as caseworker then switch to your branches and approve the draft order as a judge.

**Review by jthmcts — APPROVED** [review]
